src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx

```diff
@@ -39,27 +55,54 @@ export const FairEditorialItem: React.FC<FairEditorialItemProps> = ({
     action: ActionType.clickedArticleGroup,
   }
 
+  const image = article.thumbnailImage?.[size]!
+  const variants = {
+    large: {
+      primary: ["xl", "xxl"] as TextVariant[],
+    },
+    small: {
+      primary: ["lg", "lg", "xl"] as TextVariant[],
+    },
+  }[size]
+
+  const ItemImage: React.FC<ItemImageProps> = ({
+    width = "100%",
+    height = "100%",
+  }) => (
+    <Image
+      width={width}
+      height={height}
+      src={image.src}
+      srcSet={image.srcSet}
+      lazyLoad={true}
+      alt={article.thumbnailTitle!}
+    />
+  )
```

Comment:
Maybe it would be better to put this in a separate component and extract data using `createFragmentContainer`?